ui/src/components/pages/Namespace/index.tsx

```diff
@@ -60,7 +61,67 @@ export function Namespaces() {
       ];
     }
     if (!data) {
-      return [];
+      return [
```

Comment:
Instead of duplicating the whole array. Just use a "default" version of data in the existing array that is built. 

i.e  
let dataToUse = data;
if (!dataToUse) {
 dataToUse = defaultData; // 0s and empty arrays
}
return //existing array but using "dataToUse" instead of the original data object.

Reply:
This "default" data can be stored as a const up top and also used down on line 224

Reply:
Done!